pages/admin/permit-holder/[permitHolderId].tsx

```diff
@@ -2,94 +2,162 @@ import { GetServerSideProps } from 'next'; // Get server side props
+        addressLine2: data.applicant.addressLine2 || undefined,
+        postalCode: data.applicant.postalCode,
+        status: data.applicant.status || undefined,
+      });
+      setPermits(
+        data.applicant.permits.map(permit => ({
+          rcdPermitId: permit.rcdPermitId,
+          expiryDate: permit.expiryDate,
+          applicationId: permit.applicationId,
+          isRenewal: permit.application.isRenewal,
+          status: permit.application.applicationProcessing.status,
+        }))
+      );
+      const files: PermitHolderAttachedFile[] = [];
+      data.applicant.fileHistory.forEach(application => {
+        application.documentUrls?.forEach(documentUrl => {
+          files.push({
+            associatedApp: application.appNumber || undefined,
+            dateUploaded: application.createdAt,
+            fileUrl: documentUrl,
+          });
+        });
+      });
+      setAttachedFiles(files);
+      setMedicalHistoryData(
+        data.applicant.applications.map(application => ({
+          disablingCondition: application.disability,
+          dateUploaded: application.createdAt,
+          associatedApplication: application,
+        }))
+      );
+      setPreviousPhysicianData(
+        data.applicant.medicalHistory.map(record => ({
+          name: record.physician.name,
+          mspNumber: record.physician.mspNumber,
+          phone: record.physician.phone,
+        }))
+      );
+    },
+  });
 
   return (
     <Layout>
       <GridItem rowSpan={1} colSpan={12} marginTop={3}>
-        <PermitHolderHeader
-          applicant={applicant as unknown as Applicant}
-          applicationStatus={applicationStatus}
-        />
+        <PermitHolderHeader applicant={applicantData} applicantStatus={applicantData?.status} />
       </GridItem>
       <GridItem rowSpan={12} colSpan={5} marginTop={5} textAlign="left">
         <Stack spacing={5}>
-          <PersonalInformationCard applicant={applicant as unknown as Applicant} />
-          {/* Temporarily commented to pass CI checks for View Request page */}
-          {/* <DoctorInformationCard physician={physician} />
-          <GuardianInformationCard guardian={guardian} /> */}
+          <PersonalInformationCard applicant={applicantData} />
+          <DoctorInformationCard
+            physician={data?.applicant.medicalInformation.physician}
+            previousPhysicianData={previousPhysicianData}
+          />
+          <GuardianInformationCard guardian={data?.applicant.guardian} />
         </Stack>
       </GridItem>
 
       <GridItem rowSpan={12} colSpan={7} marginTop={5} textAlign="left">
         <Stack spacing={5}>
-          <AppHistoryCard />
-          <AttachedFilesCard />
-          <MedicalHistoryCard />
+          <AppHistoryCard permits={permits || []} />
+          <AttachedFilesCard attachedFiles={attachedFiles} />
+          <MedicalHistoryCard medicalHistory={medicalHistoryData} />
         </Stack>
       </GridItem>
```

Comment:
We would do our conditional rendering checks here, so that if any of the data is `undefined` or invalid, we don't render the card at all.

Reply:
Also, design team recently added loading states which we will address in the near future. For now, we will just have the card not render while the data is loading.